settings.yml

```diff
@@ -57,15 +57,15 @@
   
   # Either `true` to allow squash-merging pull requests, or `false` to prevent
   # squash-merging.
-  #allow_squash_merge: true
+  allow_squash_merge: false
   
   # Either `true` to allow merging pull requests with a merge commit, or `false`
   # to prevent merging pull requests with merge commits.
-  #allow_merge_commit: true
+  allow_merge_commit: true
```

Comment:
Oh interesting but I guess having the option isn't really harmful.

Reply:
It isn't... most of the time. In more complicated flows like git-flow it can add some headache when merging back and forth between release branch, master and feature, but that won't be our case I hope...

you're right, I'll add PR when have historically-proved arguments, not only opinions:)